src/sql/workbench/services/queryEditor/browser/queryEditorService.ts

```diff
@@ -99,9 +99,17 @@ export class QueryEditorService implements IQueryEditorService {
 	}
 
 	////// Private functions
+	// Gets language depending on connection provider.
+	private getLanguageFlavor(providerName?: string): string {		//TODOKusto: temporary fix until Anthony's generic fix to make it language agnostic
```

Comment:
Is this fix actively in development?

Reply:
This would be the same fix which I talked about in previous comment, fetching the languageMode using package.json and connectIonManagementService